package-parser/package_parser/commands/generate_annotations/_generate_annotations.py

```diff
@@ -0,0 +1,123 @@
+import json
+from io import TextIOWrapper
+from pathlib import Path
+from typing import Any
+
+from package_parser.commands.find_usages import (
+    ClassUsage,
+    FunctionUsage,
+    UsageStore,
+    ValueUsage,
+)
+from package_parser.commands.get_api import API
+from package_parser.utils import parent_qname
+
+
+def generate_annotations(
+    api_file: TextIOWrapper, usages_file: TextIOWrapper, out_dir: Path
+):
```

Comment:
Um die Konsistenz mit #433 und #436 zu erhalten sollten hier anstatt TextIOWrapper Strings als Parameter verwendet werden. 
Da der Aufruf in #436 Pfadangaben vom Typ String übergibt, könntet ihr diese dann 'intern' zu TextIOWrappern umwandeln.

Reply:
Wir haben uns hier an der Struktur von anderen Modulen und dem Code orientiert, den Lars uns gezeigt hat. Die eigentliche Funktion ist die _determine_constant_parameters() Methode weiter unten. Da verwenden wir allerdings die Datenstruktur die im Rest des Projektes verwendet wird.
Die Funktion generate_annotations ist Vorarbeit für die Zukunft. Der Plan ist es dort nächsten Freitag die Ergebnisse von #433 und #436 zu integrieren.
Wenn ihr Wert darauf legt kann ich es allerdings erstmal mit einem String als Parameter lösen.